modules/claims_api/spec/requests/v1/intent_to_file_request_spec.rb

```diff
@@ -61,12 +61,12 @@
         end
       end
 
```

Comment:
Bgs throws 'Server' here, so should be throw a 404 or a 422? I know previously we voted for 422, but now I am thinking that a 422 is most likely due to a client action, and 404 is a server action. thoughts?

Reply:
Sorry for the delayed response. I feel like 404 implies something in the request path is incorrect. If we're hitting a valid server endpoint with a correctly-formed URL, then we should throw a 422. Just my opinion, I could be wrong.

Reply:
Since these questions 2 weeks ago, and digging into this more, I have changed this PR, and learned more.

Regarding `requests/v1/intent_to_file_request_spec.rb:64 `, the specific `fault_string` includes "insertIntentToFile-->Veteran with ID 1521881879 not found in database", which is a 404, 'Not Found'.